tests/rest/v1/accounts.spec.ts

```diff
@@ -168,6 +169,22 @@ describe("account", () => {
     });
   });
 
+  it("mutes by ID for 1s", () => {
+    return sessions.use(2, async ([alice, bob]) => {
+      let relationship = await alice.rest.v1.accounts
+        .$select(bob.id)
+        .mute({ duration: 1 });
+      expect(relationship.muting).toBe(true);
+
+      await sleep(5000);
```

Comment:
I set a bit longer 5 seconds sleep here since when I tested with mastodon.social, it takes 2-3 seconds for the latest state applied. It might be able to be shorter but it may cause flaky tests. Do you think 5 seconds of sleep is acceptable? I can find only 1 second sleep in any test.

Reply:
I have no idea about how Mastodon servers undertakes caching and job handling, but it's so much better than shipping without testing anything. So let me merge it and see how it turns out.

In the case if it undermined the test stability, I will increase the sleep seconds or even comment it out.

Reply:
OK, that sounds good.👍🏻  There was no issue at least when I tested on my repo.